envs/monkey_zoo/blackbox/conftest.py

```diff
@@ -32,6 +32,11 @@ def no_gcp(request):
     return request.config.getoption("--no-gcp")
 
 
+@pytest.fixture(scope="session")
+def machines_to_start(request):
+    return request.config.getoption("-k")
```

Comment:
What if I do `-k depth_1 or depth_1`?  or `-k depth`?

The fix for this would be to associate certain machines to certain tests. Feel free to either fix it or create an issue to add this feature, it's a good one. But this implementation will break the `-k` option.

Reply:
#2141